src/Microsoft.HttpRepl.Fakes/MockHttpContent.cs

```diff
@@ -26,7 +26,7 @@ protected async override Task SerializeToStreamAsync(Stream stream, TransportCon
 
         protected override bool TryComputeLength(out long length)
         {
-            length = Content.Length;
+            length = Content?.Length ?? 0;
```

Comment:
Is Content allowed to be null?  What's the difference between null and string.empty?
(In other words, could this also be enforced in the constructor with `Content = content ?? string.empty;`)

Reply:
Well that was an interesting rabbit hole. It looks like `Content` could sort of be null, because if `TryComputeLength` returned zero, then it wouldn't even try to call `SerializeToStreamAsync`. I'm not sure if that's guaranteed though, so we shouldn't rely on it. And making the change as you suggested, to just change `null` to `string.Empty` works the same way, but is more safe.